src/deepsparse/v2/schedulers/scheduler_group.py

```diff
@@ -38,6 +38,7 @@ def submit(
         self,
         *args,
         operator: Operator,
+        loop: Any = None,
```

Comment:
Not sure what version of python it was introduced in, but I believe `asyncio.AbstractEventLoop` is the semi-generic type you'd want here
```suggestion
        loop: Optional[AbstractEventLoop] = None,
```

You'd need to add a couple of imports too